model/device.go

```diff
@@ -39,6 +39,11 @@ const (
 	AttrNameCreated = "created_ts"
 )
 
+const (
+	runeDollar = '\uFF04'
+	runeDot    = '\uFF0E'
```

Comment:
I was going to ask if this is right, but I see they're just the full width variants ． ＄.

Reply:
yep